app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::BaseController < ActionController::Base
+  before_action :require_login!
+  helper_method :current_person
+
+  def require_login!
+    return true if authenticate_token
+    if token_exists
+      render json: { errors: [{ detail: 'Your key is expired' }] }, status: 401
```

Comment:
why this details needed?

Reply:
is there no need in expiration of a token?

Reply:
@igorkasyanchuk ^^